app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/state/RealVpnStateMonitor.kt

```diff
@@ -105,13 +105,13 @@ class RealVpnStateMonitor @Inject constructor(
             SELF_STOP -> VpnStopReason.SELF_STOP
             REVOKED -> VpnStopReason.REVOKED
             ERROR -> VpnStopReason.ERROR
+            SNOOZED -> VpnStopReason.SNOOZED()
```

Comment:
This is weird, what does it mean snoozed without any `snoozedTriggerAtMillis`?

Reply:
@aitorvs VpnStoppingReason is unfortunately stored in the DB and since I don’t want to modify that I did the line of code in question. Maybe this will be fixed by removing SNOOZED and replacing it with SELF_STOP(snoozetime).